.github/workflows/reusable-tox.yml

```
@@ -292,6 +292,8 @@ jobs:
         --quiet
         --
         python -Im pre_commit install-hooks
+    # Create GHA Job Summary markdown table of the coverage report
+    # For details: ../../tox.ini '[testenv]' 'commands_post'
```

This workflow is generic. That's only happening when the env with pytest is being invoked, but not the other envs.

---

```suggestion
    # But only for 'pytest' env in 'tox'.
    # For details: ../../tox.ini '[testenv]' 'commands_post'
```
Something like that?

---

Technically, for both `[testenv]` and `[testenv:pytest]` which are currently set up equivalently.